app/src/main/java/com/blogspot/e_kanivets/moneytracker/controller/PeriodController.java

```diff
@@ -133,15 +133,15 @@ public Period yearPeriod() {
     public Period allTimePeriod() {
         Calendar cal = Calendar.getInstance();
 
-        // set start of time by Jesus
+        // set start of date by Jesus
```

Comment:
Auto-renaming, I believe. Not sure you needed to change it. Review all comments in this PR's code, please.